openmetadata-ui/src/main/resources/ui/src/pages/signup/index.test.tsx

```diff
@@ -0,0 +1,331 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { act, fireEvent, render } from '@testing-library/react';
+import React, { ReactNode } from 'react';
+import Signup from '.';
+import * as AppState from '../../AppState';
+import { createUser } from '../../axiosAPIs/userAPI';
+import { getImages } from '../../utils/CommonUtils';
+
+const mockChangeHandler = jest.fn();
+const mockSubmitHandler = jest.fn();
+const mockShowErrorToast = jest.fn();
+
+const mockCreateUser = {
+  data: {
+    id: '911d4be4-6ebf-48a0-9016-43a2cf716428',
+    name: 'test.user',
+    fullyQualifiedName: 'test.user',
+    displayName: 'Test User',
+    version: 0.1,
+    updatedAt: 1665145804919,
+    updatedBy: 'test.user',
+    email: 'test.user@test.com',
+    href: 'http://sandbox-beta.open-metadata.org/api/v1/users/911d4be4-6ebf-48a0-9016-43a2cf716428',
+    isAdmin: false,
+    profile: {
+      images: {
+        image:
+          'https://lh3.googleusercontent.com/a/ALm5wu0HwEPhAbyRha16cUHrEum-zxTDzj6KZiqYsT5Y=s96-c',
+        image24:
+          'https://lh3.googleusercontent.com/a/ALm5wu0HwEPhAbyRha16cUHrEum-zxTDzj6KZiqYsT5Y=s24-c',
+        image32:
+          'https://lh3.googleusercontent.com/a/ALm5wu0HwEPhAbyRha16cUHrEum-zxTDzj6KZiqYsT5Y=s32-c',
+        image48:
+          'https://lh3.googleusercontent.com/a/ALm5wu0HwEPhAbyRha16cUHrEum-zxTDzj6KZiqYsT5Y=s48-c',
+        image72:
+          'https://lh3.googleusercontent.com/a/ALm5wu0HwEPhAbyRha16cUHrEum-zxTDzj6KZiqYsT5Y=s72-c',
+        image192:
+          'https://lh3.googleusercontent.com/a/ALm5wu0HwEPhAbyRha16cUHrEum-zxTDzj6KZiqYsT5Y=s192-c',
+        image512:
```

Comment:
lets move this to separate file in mock folder

Reply:
Why do we need to mention all the images? We can just have it like below and it should work
```
image: ""
```

Reply:
Done ✅